extension/docs/HowToRun.md

```diff
@@ -125,19 +129,24 @@ Other configurations can be set as direct child attributes in `ADYEN_INTEGRATION
 {
   "commercetools": {...},
   "adyen": {...},
+  "paymentMethodsToNamesConfig": {
```

Comment:
`paymentMethodsToNamesConfig` doesn't provide any hint about the context that this applies. It would be better to be explicit on this attribute when and where this is applicable. Renaming this variable can fix this issue.

Reply:
Do you have any naming suggestions?

Reply:
how about `adyenPaymentMethodNameMappings`?

Reply:
https://github.com/commercetools/commercetools-adyen-integration/pull/699/commits/91586bd93100923b362fcb3d28b64779038d2ad8